module/trace/constants.go

```diff
@@ -200,11 +200,7 @@ const (
 	FVMEnvRemoveAccountContractCode SpanName = "fvm.env.removeAccountContractCode"
 	FVMEnvGetSigningAccounts        SpanName = "fvm.env.getSigningAccounts"
 
-	FVMCadenceParseProgram     SpanName = "fvm.cadence.parseProgram"
-	FVMCadenceCheckProgram     SpanName = "fvm.cadence.checkProgram"
-	FVMCadenceInterpretProgram SpanName = "fvm.cadence.interpretProgram"
-	FVMCadenceEncodeValue      SpanName = "fvm.cadence.encodeValue"
-	FVMCadenceDecodeValue      SpanName = "fvm.cadence.decodeValue"
```

Comment:
would it make sense to still keep these as `FVMCadenceDecodeValue  SpanName = "decodeValue"`?